### PR TITLE
Prep up for NPM

### PR DESCRIPTION
**Pull request by knowler**
* ~Use Babel to compile since we’re using ES6 syntax~
* Rename main script to `index.js`
* Move `utils/camelCase.js` to `utils.js` and use named export
* Add `.npmignore` so we don’t publish unnecessary files/directories
* Add [np](https://github.com/sindresorhus/np) since publishing to NPM is error prone (publish whats in your current directory — doesn’t build or clean first). np also handles GitHub tagging/versioning/releasing.

We can squash these.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ import camelCase from './utils/camelCase';
  * The routing fires all common scripts, followed by the page specific scripts.
  * Add additional events for more control over timing e.g. a finalize event
  */
-class Router {
+export default class {
```
Comment (austinpray) [inline index.js]:
```suggestion
export class Router {
```

**Review by austinpray — CHANGES_REQUESTED** [review]
named exports are better

File: index.js (hunk 1 above)
Comment (knowler) [reply index.js]: Cool, I agree.

**Comment by QWp6t** [comment]
we won't need .npmignore after rewrite

**Comment by knowler** [comment]
@QWp6t I guess the only thing that would matter after the rewrite is `np`. Do we want to use that for releases/versioning and stuff?

**Comment by Log1x** [comment]
I'd say roll a GitHub action for it instead.

https://github.com/actions/npm

**Comment by QWp6t** [comment]
I've never used a GitHub action, but [I have used travis to handle deployments](https://github.com/QWp6t/browsersync-webpack-plugin/blob/master/.travis.yml#L9-L16) when a new release gets tagged.

**Comment by knowler** [comment]
Travis or a GitHub Action seem more foolproof to me. Closing this.

**Pull request closed without merging**